cmd/dependabot/internal/cmd/update.go

```diff
@@ -238,7 +245,7 @@ func readInputFile(file string) (*model.Input, error) {
 	return &input, nil
 }
 
-func processInput(input *model.Input) {
+func processInput(input *model.Input, flags *UpdateFlags) error {
```

Comment:
I'm not super up on my GoLang, but wouldn't this be `*error` because we return `nil`?

Reply:
`error` is an interface, which can be `nil` so that's not a problem here. However I noticed this function only returns `nil` so we don't need a return value at all.

Reply:
I removed the return type